src/Sdk/Coverage/Coverage.ts

```diff
@@ -33,7 +31,7 @@ export default class CoverageSdk {
         return response.payload;
     }
 
-    async get(id: CoverageId, includeDeleted = false): Promise<Coverage> {
+    async get(id: Coverage['uuid'] | Coverage['id'], includeDeleted = false): Promise<Coverage> {
```

Comment:
Why not leaving the `CoverageId` type and setting it to `Coverage['uuid'] | Coverage['id']`, to make things more streamlined?

Reply:
And I guess same could be done for other types. This way, if we then decide to completely get rid of referencing by `id`, we would only need to update a single type.

Reply:
My thinking was that `CoverageId` naming is confusing, as it refers to UUID or ID; with UUID being preferred. So I have decided to just inline the union type to make it maximum obvious which values it accepts.

Do you still think I should reintroduce `CoverageId` type here?

Reply:
I think it will make the maintaining of the types easier for sure.
Agree with the naming part, maybe call that `CoverageReference`? That's generic enough and still communicates the purpose of the type. (Though the argument is still called `id`, so maybe `CoverageId` is still OK 😁)

Reply:
Alright, I've re-introduced the `*Id` types. I think it's more straighforward than `CoverageReference`.

Thanks for your comments Oleg :+1:

ba8086e0b17793a4d6b4f09b60e721bb2fdc512b